ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.9:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.9
+binary_build_commands: go build .
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+canonical_go_repository: github.com/IBM-Cloud/cloud-provider-ibm
+images:
+- dockerfile_path: openshift-hack/images/Dockerfile.openshift
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.9:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9
+  to: ibm-cloud-controller-manager
+promotion:
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
```

Comment:
Not really sure what the scope of this should be, but we have a full unit test and fvt that could be run here as well

Reply:
Yep, I added unit tests and fmt. fvt requires an additional binary, so I'd prefer to skip this job. 
I think that what we have now is enough for a downstream fork. Later we will add an e2e job for IBM Cloid.